blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/EventsIO.java

```diff
@@ -20,6 +20,6 @@
 import java.util.Map;
 
 public interface EventsIO {
-    public void insert(String tenant, List<Map<String, Object>> metrics) throws Exception;
-    public List<Map<String, Object>> search(String tenant, Map<String, List<String>> query) throws Exception;
+    void insert(String tenantId, Map<String, Object> event) throws Exception;
```

Comment:
It looks like you've changed the second parameter here from a list of maps, to a map.  any particular reason?  Isn't this api exposed to the users?

Reply:
https://github.com/rackerlabs/blueflood/blob/master/blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpEventsIngestionHandler.java#L69
At this line, we are deserializing request body into Event instance, and it accepts only one event object. There is no array here. However, we are wrapping that event object into a collection which will always have only one object. So, there was unnecessary wrapping around one object. I just removed it and made corresponding changes.